SCRUM-12 quiz repository implementation

OktayRasimov opened the pull request:
### Description 
-Implemented basic quiz repo with crud methods
-inmemory quizes as of now till db connection

### Dependencies
This pr is stacked on #6 
Please review #6  before this

### Related Jira: [SCRUM-11]
Jira link: https://react-kotlin.atlassian.net/jira/software/projects/SCRUM/boards/1/backlog?selectedIssue=SCRUM-12

OktayRasimov committed:
implement quiz repository with basic crud imeplementations

RomaMocherniuk commented on src/main/kotlin/com/reactkotlin/quiz/backend/repository/QuizRepositoryImpl.kt:
```diff
@@ -0,0 +1,37 @@
+import org.springframework.stereotype.Repository
+
+@Repository
+class InMemoryQuizRepository : QuizRepository {
```
I think `InMemoryQuizRepository` should be responsible for creating a quiz `id` when we add a new quiz
Later, `Id` will be created by DB

OktayRasimov committed:
SCRUM-11-create initial quiz model (#6)

add initial quiz model with counter tracker,variables for spring security,test security config
